src/Features/Core/Portable/Structure/BlockStructureOptions.cs

```diff
@@ -11,6 +11,9 @@ namespace Microsoft.CodeAnalysis.Structure
         bool ShowOutliningForCommentsAndPreprocessorRegions = true,
         bool ShowOutliningForDeclarationLevelConstructs = true,
         bool ShowOutliningForCodeLevelConstructs = true,
+        bool CollapseRegionsWhenFirstOpened = true,
+        bool CollapseImportsWhenFirstOpened = false,
```

Comment:
interesting.  is that to match VB current bheavior?

Reply:
This matches the current VB and C# behaviour. Before this PR we didn't collapse usings, nor have an option for it.